### ISSUE TITLE
Problems with installing

### ISSUE THREAD

**Issue by Vendelieu** [gliderlabs/herokuish#451]
i've go this issue when i installing - http://dokku.viewdocs.io/dokku/getting-started/advanced-installation/#custom-herokuish-build
```
go: downloading github.com/jteeuwen/go-bindata v3.0.7+incompatible
go: extracting github.com/jteeuwen/go-bindata v3.0.7+incompatible
go get github.com/jteeuwen/go-bindata/...: no matching versions for query "latest"
Makefile:52: recipe for target 'deps' failed
make: *** [deps] Error 1
Makefile:38: recipe for target 'build-in-docker' failed
make[2]: *** [build-in-docker] Error 2
make[2]: Leaving directory '/tmp/herokuish'
Makefile:166: recipe for target 'stack' failed
make[1]: *** [stack] Error 2
make[1]: Leaving directory '/root/dokku'
Makefile:123: recipe for target 'dependencies' failed
make: *** [dependencies] Error 2
```
all console log:
https://pastebin.com/JC8jXLPh

**Comment by mlandauer** [comment]
I'm seeing this same error when building herokuish with `make build-in-docker`.

```
...
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro \
		-v /var/lib/docker:/var/lib/docker \
		-v /Users/matthew/git/oaf/herokuish:/src/github.com/gliderlabs/herokuish -w /src/github.com/gliderlabs/herokuish \
		-e IMAGE_NAME=herokuish -e BUILD_TAG=dev -e VERSION=master \
		herokuish-build make -e deps build
docker pull heroku/heroku:18-build
18-build: Pulling from heroku/heroku
Digest: sha256:5161da7713dafaaa2a1d2e5cc22f5a2f833c22f6b894bfd9598e8526fb4c0f84
Status: Image is up to date for heroku/heroku:18-build
go get -u github.com/jteeuwen/go-bindata/...
go: finding github.com/progrium/go-basher v0.0.0-20170201215011-ad5de635edd1
go: finding github.com/mitchellh/go-homedir v1.1.0
go: finding github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
go: finding gopkg.in/yaml.v2 v2.2.2
go: finding gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405
go: finding github.com/jteeuwen/go-bindata/... latest
go: finding github.com/jteeuwen/go-bindata v3.0.7+incompatible
go: downloading github.com/jteeuwen/go-bindata v3.0.7+incompatible
go: extracting github.com/jteeuwen/go-bindata v3.0.7+incompatible
go get github.com/jteeuwen/go-bindata/...: no matching versions for query "latest"
Makefile:52: recipe for target 'deps' failed
make: *** [deps] Error 1
make: *** [build-in-docker] Error 2
```

This looks to be a [go issue](https://github.com/golang/go/issues/27215) related to the introduction of modules. The fix for it will land in Go 1.13 but until then the following change to the `Makefile` does the trick as a workaround:

```
deps:
	docker pull heroku/heroku:18-build
	go get -u github.com/jteeuwen/go-bindata/
	go get -u github.com/progrium/gh-release/
	go get -u github.com/progrium/basht/
	go get -u github.com/jteeuwen/go-bindata/...
	go get -u github.com/progrium/gh-release/...
	go get -u github.com/progrium/basht/...
	go get || true
```

**Comment by josegonzalez** [comment]
This should really be filed at the Dokku project. If the docs in this repository fail to build out a custom herokuish build, then we can fix it here, but as is, it sounds like the Dokku docs are a bit wrong.
